### PR TITLE
When non-staffers try to log into staff computer, explain problem in error message

### PR DESCRIPTION
**Pull request by kpengboy**
The error message tells them they aren't a staffer.

### CODE REVIEW

**Comment by kpengboy** [comment]
Any comments about the message layout or coding style?

File: modules/ocf_desktop/files/xsession/lightdm/lightdm-gtk-greeter-staff.po [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#: ../src/lightdm-gtk-greeter.c:1479
+msgid "Incorrect password, please try again"
+msgstr "Incorrect username or password,\n"
+"or you're not a staffer."
```
Comment (chriskuehl) [inline modules/ocf_desktop/files/xsession/lightdm/lightdm-gtk-greeter-staff.po]: Can we be more explicit about this? Something like...

"Sorry, but only OCF volunteer staffers can log in to this computer. Please use a different computer instead (or if you actually are staff, check your password)."

Anyone trying to log in to the computer is already clearly not very attentive, so I think making the message slightly longer and more direct would be a plus.

File: modules/ocf_desktop/manifests/xsession.pp [hunk 2]
```diff
@@ -72,17 +72,25 @@
...
 
+  $po = $::staff_only ? {
+    true    => 'lightdm-gtk-greeter-staff.po',
+    default => 'lightdm-gtk-greeter.po',
+  }
+
+  file {
+    "/opt/share/xsession/${po}":
+      source  => "puppet:///modules/ocf_desktop/xsession/lightdm/${po}";
+  }
```
Comment (chriskuehl) [inline modules/ocf_desktop/manifests/xsession.pp]: nit: first lines can be combined like

```puppet
file { "/opt/share/xsession/${po}":
  source  => "puppet:///modules/ocf_desktop/xsession/lightdm/${po}";
}
```
totally optional

File: modules/ocf_desktop/manifests/xsession.pp [hunk 3]
```diff
@@ -72,17 +72,25 @@
...
       mode    => '0755',
       source  => 'puppet:///modules/ocf_desktop/xsession/lightdm/session-cleanup';
-    '/opt/share/xsession/lightdm-gtk-greeter.po':
-      source  => 'puppet:///modules/ocf_desktop/xsession/lightdm/lightdm-gtk-greeter.po';
   }
 
   # overwrite greeter strings with OCF ones
   package {'gettext':;}
 
+  $po = $::staff_only ? {
```
Comment (chriskuehl) [inline modules/ocf_desktop/manifests/xsession.pp]: at some point, we should either rename this to `staffOnly` or rename `skipNetworking` and friends to snake case
(but not related to your PR. i'll ticket)

**Comment by chriskuehl** [comment]
looks good, ship it (optionally make the message more explicit?)

File: modules/ocf_desktop/manifests/xsession.pp (hunk 3 above)
Comment (chriskuehl) [inline modules/ocf_desktop/manifests/xsession.pp]: rt#4146

**Comment by chriskuehl** [comment]
lg2m

**Pull request merged**